frontend/catalyst/pennylane_extensions.py

```diff
@@ -234,7 +282,7 @@ def grad(f, *, method=None, h=None, argnum=None):
             - ``"adj"`` represents the adjoint differentiation method.
 
         h (float): the step-size value for the finite-difference (``"fd"``) method
-        argnum (int, List(int)): the argument indices to differentiate
+        argnum Tuple[int, List[int]]: the argument indices to differentiate
```

Comment:
I just forgot to put the parenthesis, thanks. They are due to a pattern I read in some internal coding style document. As far as I know, automatic typecheckers only honor the specifications that are made in code, so this is not a strict format. I only use this style because I know it has precise meaning elsewhere :)